### PR TITLE
Feature/search page

### LINKED ISSUE #29: Create a search page
Create a serch page. 
Use a folder structure that was accepted at the meeting

### PR DESCRIPTION
**Pull request by Pashkaua**
Create serch page (modal windows)
add redux(action, reducer), react-hook-forms.
Home page - create putton, just for test.

### CODE REVIEW

**Review by ivan-bonk — APPROVED** [review]
Nice work!
As i see you import modal window in a Home page
It's not a problem, but in out design we have this button in a Result page

**Comment by Pashkaua** [comment]
> Nice work!
> As i see you import modal window in a Home page
> It's not a problem, but in out design we have this button in a Result page

I think the button is in a  home page, but we need  to discuss this at a meeting on friday.

File: src/pages/search/components/modalBody.component.tsx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+import { searchAction } from '../../../store/actions/search.action';
+import { SearchInterface } from '../search.interfaces';
+
+export const ModalBody: React.FC<SearchInterface> = (props: SearchInterface) => {
+  const { register, handleSubmit, errors } = useForm();
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  const onSubmit = (data: {}): void => {
+    console.log(data);
```
Comment (okhabr) [inline src/pages/search/components/modalBody.component.tsx]: Would you mind removing this console.log ?

**Review by okhabr — APPROVED** [review]
Good job :) 
Left one minor comment

File: src/pages/search/components/modalBody.component.tsx (hunk 1 above)
Comment (Pashkaua) [reply src/pages/search/components/modalBody.component.tsx]: )))
classic bug ...
yes, of course I will romove .

**Review by Danik1702 — APPROVED** [review]
Looks fine! : )

**Comment by Pashkaua** [comment]
fix conflicts

**Pull request merged**